azerothcore/azerothcore-wotlk
# fix(DB/Creature) Remove Fire Immunity from Warlock's Infernal (Inferno Spell)

## Changes Proposed:
-  Removed `spell_school_immune_mask` from Warlock's [Infernal](https://wotlkdb.com/?npc=89)
-  [Infernal](https://wotlkdb.com/?npc=89) doesn't need it since it already have [Immunity:Fire](https://wotlkdb.com/?spell=7942) spell in `creature_addon`
- After removing `spell_school_immune_mask` [Immolation](https://wotlkdb.com/?spell=19483) spell started triggering from [Infernal](https://wotlkdb.com/?npc=89)

## SOURCE:

https://youtu.be/dPORXIbZKkw?t=80

## Tests Performed:

- Checked [Immolation](https://wotlkdb.com/?spell=19483) triggering
- Summoned [Infernal](https://wotlkdb.com/?npc=89) through `.npc add 89` and tested fire spells against him


## How to Test the Changes:


1. Cast [Inferno](https://wotlkdb.com/?spell=1122) at any mob/pack
2. Check for [Immolation](https://wotlkdb.com/?spell=19483) to trigger
3. `.npc add 89`
4. Try to cast fire spells on him



## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.
